### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by rpgayatri**
1) Refactoring of item creation and fee creation.
2) Requires latest qwanda.
3) Removed job id from new load slack message.

### CODE REVIEW

File: src/main/java/life/genny/rules/QRules.java [hunk 1]
```diff
@@ -8002,5 +8007,111 @@ public void publishViewCmdMessage(final String viewType, final String rootCode)
...
+
+				/* Not mandatory */
+				String begDescription = loadBe.getValue("PRI_DESCRIPTION", null);
+
+				/* get item name */
+				String paymentsItemName = PaymentUtils.getPaymentsItemName(loadBe, begBe);
+				println("payments item name ::"+paymentsItemName);
+			
+				 /* driverPriceIncGST = ownerPriceIncGST.subtract(feePriceIncGST) */
+				Money ownerAmountWithoutFee = offerBe.getValue("PRI_OFFER_DRIVER_PRICE_INC_GST", null);
```
Comment (DevJS) [inline src/main/java/life/genny/rules/QRules.java]: There is an issue sometimes where this price is not calculated. so this could be null!

Comment (rpgayatri) [reply src/main/java/life/genny/rules/QRules.java]: If its null, its handled when the constructor is created (which is the next line of code). Null values throw exception.

Comment (rpgayatri) [reply src/main/java/life/genny/rules/QRules.java]: Thats a good point. I will make some appropriate changes.

**Review by DevJS — APPROVED** [review]
looking good

**Pull request merged**